pkg/user/tx_client.go

```diff
@@ -40,7 +40,9 @@ import (
 const (
 	DefaultPollTime          = 3 * time.Second
 	txTrackerPruningInterval = 10 * time.Minute
-	evictionPollTime         = 1 * time.Minute
+	// evictionPollTime is the polling interval for checking if an evicted transaction
+	// gets committed after experiencing a broadcast error during resubmission
+	evictionPollTime = 1 * time.Minute
```

Comment:
[question] is this really a polling _interval_ or a polling _timeout_? Based on its usage in the previous PR, I think it's a just timeout. In my mind: interval means once every minute. Timeout means once after one minute.

Ref: https://github.com/celestiaorg/celestia-app/blob/12414a9e89ff7e909219f82634d34ce834adfe7d/pkg/user/tx_client.go#L628-L630

Reply:
it is timeout! you're right! I'll change it now